index.html

```diff
@@ -165,7 +165,51 @@ <h1 class="crete w400 s72 lh88 ls037 orange">About me</h1>
                 </li>
             </ul>
         </section>
-        
+        <section id="contact">
+            <p class="inter nBlue2 s32 w700 lh44">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd 
+               like to<br>chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xlevqbpl" method="post">
```

Comment:
- Kindly make sure your form is submitting, I think it's because of the the structure of your form that I pointed earler about removing the `ul` and `li` :recycle: